src/platform/plugins/shared/dashboard/public/dashboard_api/get_dashboard_api.ts

```diff
@@ -164,10 +164,6 @@ export function getDashboardApi({
     },
     isEmbeddedExternally: Boolean(creationOptions?.isEmbeddedExternally),
     isManaged,
-    reload$: merge(
-      unifiedSearchManager.internalApi.controlGroupReload$,
-      unifiedSearchManager.internalApi.panelsReload$
-    ).pipe(debounceTime(0)),
```

Comment:
The `debounceTime` was causing a race condition where the Dashboard's `reload$` was firing **after** each embeddable reloaded - this meant that each embeddable was receiving a stale search session ID. After further digging, it was determined that `controlGroupReload$` was legacy code that wasn't necessary anymore - so, I removed it + cleaned up `panelsReload$`